acceptance/config/aio/options.rb

```diff
@@ -1,9 +1,3 @@
 {
   :type => 'aio',
-  :pre_suite => [
-    'setup/common/pre-suite/000-delete-puppet-when-none.rb',
-    'setup/aio/pre-suite/010_Install.rb',
-    'setup/aio/pre-suite/021_InstallAristaModule.rb',
-    'setup/aio/pre-suite/022_Remove_LD_PRELOAD.rb',
```

Comment:
I'm not seeing this step in beaker-puppet. Should we add it there?

Reply:
@MikaelSmith and I decided we probably don't need it anymore, but I'm happy to add it to beaker-puppet if that was misguided

Reply:
👍 this passes on `ciscoxr-64a`